sys/shell/commands/sc_netif.c

```diff
@@ -36,6 +36,7 @@
 #include "net/gnrc/pktbuf.h"
 #include "net/gnrc/netif/hdr.h"
 #include "net/gnrc/sixlowpan/netif.h"
+#include "net/netdev2/ieee802154.h"
```

Comment:
The more of these really specific commands come in the more I start thinking that this might be a good idea.